src/Scripting/CSharp/project.json

```diff
@@ -5,8 +5,8 @@
     "System.Dynamic.Runtime": "4.0.10"
   },
   "frameworks": {
-    "dotnet5.4": {
-      "imports": "portable-net452"
+    "netstandard1.3": {
+      "imports": [ "portable-net452", "dotnet" ]
```

Comment:
Importing dotnet without a version is the same as dotnet0.0. We should however try to keep these aligned so we should do dotnet5.4 here which is the precursor to netstandard1.3